scripts/promise_batch_imports.py

```diff
@@ -52,7 +69,11 @@ def map_book_to_olbook(book, promise_id):
 def batch_import(promise_id):
     url = "https://archive.org/download/"
     date = promise_id.split("_")[-1]
-    books = requests.get(f"{url}{promise_id}/DailyPallets__{date}.json").json()
+    try:
+        books = requests.get(f"{url}{promise_id}/DailyPallets__{date}.json").json()
+    except MissingDailyPalletsJSON as e:
```

Comment:
needs to be raw Exception and we need to somehow account or handle as something more specific

Reply:
@cdrini if you have any bandwidth to advise with this one, I'm trying to push forward with the cron portion.

Reply:
Dropping this error catch for now.